speech/grpc/bin/speech-sample-async.sh

```diff
@@ -15,4 +15,4 @@
 
 SRC_DIR=$(cd "$(dirname "$0")/.."; pwd)
 java -cp ${SRC_DIR}/target/grpc-sample-1.0-jar-with-dependencies.jar \
```

Comment:
is SRC_DIR necessary?

`java -cp target/grpc-sample-1.0-jar-with-dependencies.jar \` perhaps?

Or even:
mvn exec:java -PAsyncRecognizeClient